### PR TITLE
Translate first 6 windows pages in Brazilian Portuguese

### PR DESCRIPTION
**Pull request by schneiderl**
Translated first six windows pages in Portuguese. 
List: 
cls.md
cmd.md
dir.md
mkdir.md
print.md
type.md

### CODE REVIEW

**Comment by schneiderl** [comment]
Hi @sbrl 

According to the Contributing Guidelines for translation introduced on #2703 language codes should follow BCP 47, just as it is with the files included on this PR.

**Comment by agnivade** [comment]
Yes, language code is correct. Will leave it to @waldyrious to review.

File: pages.pt-BR/windows/cls.md [hunk 1]
```diff
@@ -0,0 +1,7 @@
+# cls
+
+> Limpa a tela de saída.
+
+- Limpa a tela:
```
Comment (waldyrious) [inline pages.pt-BR/windows/cls.md]: Please use infinitive as in the other pages: "Limpar". (Same in the main description above)

File: pages.pt-BR/windows/dir.md [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+
+-  Mostra o conteúdo do diretório no caminho provido pelo usuário:
+
+`dir {{caminho/para/diretório}}`
+
+- Mostra o conteúdo do diretório atual, incluindo arquivos e pastas escondidas:
+
+`dir /A`
+
+- Mostra o conteúdo do diretório no caminho provido pelo usuário, incluindo arquivos e pastas escondidas:
```
Comment (waldyrious) [inline pages.pt-BR/windows/dir.md]: Perhaps simpler:
```suggestion
- Mostra o conteúdo do diretório provido pelo usuário, incluindo arquivos e pastas escondidas:
```

File: pages.pt-BR/windows/dir.md [hunk 3]
```diff
@@ -0,0 +1,19 @@
+# dir
+
+> Lista os conteúdos de um diretório.
```
Comment (waldyrious) [inline pages.pt-BR/windows/dir.md]: Make sure to use the infinitive form in all verbs for all pages (here, for example: "Lista" --> "Listar", and below: "Mostra" --> "Mostrar").

**Comment by waldyrious** [comment]
Sorry guys, I've been going through some pretty busy times lately.

@schneiderl, many thanks for the PR! I've added some comments inline, but overall this looks pretty good! Should be ready to merge once the comments are resolved :)

File: pages.pt-BR/windows/dir.md (hunk 2 above)
Comment (waldyrious) [reply pages.pt-BR/windows/dir.md]: Or even shorter (mimicking what's used for `dir.md`):
```suggestion
- Mostra o conteúdo de um diretório, incluindo arquivos e pastas escondidas:
```

File: pages.pt-BR/windows/mkdir.md [hunk 4]
```diff
@@ -0,0 +1,11 @@
+# mkdir
+
+> Cria um diretório.
+
+- Criar diretório:
```
Comment (waldyrious) [inline pages.pt-BR/windows/mkdir.md]:
```suggestion
- Criar um diretório:
```

File: pages.pt-BR/windows/mkdir.md [hunk 5]
```diff
@@ -0,0 +1,11 @@
+# mkdir
+
+> Criar um diretório.
+
+- Criar um diretório:
+
+`mkdir {{nome_do_diretorio}}`
+
+- Recursivamente cria uma árvore de diretórios aninhados:
```
Comment (waldyrious) [inline pages.pt-BR/windows/mkdir.md]: Talvez:

```suggestion
- Criar recursivamente uma árvore de diretórios aninhados:
```

**Comment by waldyrious** [comment]
One last nitpick :)

**Comment by schneiderl** [comment]
> One last nitpick :)

Oh, did not saw that. 
Thank you @waldyrious

**Comment by waldyrious** [comment]
Great, thanks so much for the PR and quick updates :) Given that there's already been two other approvals, I'll go ahead and merge.

**Pull request merged**